src/main/kotlin/com/terraformation/backend/documentproducer/db/VariableWorkflowStore.kt

```diff
@@ -28,6 +36,40 @@ class VariableWorkflowStore(
     }
   }
 
+  fun update(
+      projectId: ProjectId,
+      variableId: VariableId,
+      status: VariableWorkflowStatus,
+      feedback: String?,
+      internalComment: String?,
+  ) {
+    requirePermissions { updateInternalVariableWorkflowDetails(projectId) }
+
+    if (!dslContext.fetchExists(PROJECTS, PROJECTS.ID.eq(projectId))) {
+      throw ProjectNotFoundException(projectId)
+    }
+
+    if (!dslContext.fetchExists(VARIABLES, VARIABLES.ID.eq(variableId))) {
+      throw VariableNotFoundException(variableId)
+    }
+
+    with(VARIABLE_WORKFLOW_HISTORY) {
+      dslContext
+          .insertInto(VARIABLE_WORKFLOW_HISTORY)
```

Comment:
Is this preferred over `this`?

Reply:
I slightly prefer using the table name here just because it reads a little more like a real SQL statement, but I haven't felt strongly enough about it either way to want to propose making it a formal coding convention. I think it's fine either way.